04-Computing_Systems.Rmd

```diff
@@ -278,24 +277,24 @@ See [here](https://cs.wmich.edu/~elise/courses/cs626/s09/hussein/Parallel_and_Cl
 
 In a [computing grid](https://hazelcast.com/glossary/grid-computing/) are often **different** types of computers in **different** locations work towards an overall common goal by performing **different** tasks [@grid]. 
 
-Again, just like computer clusters, there are many types of architectures that can be rather simple to very complex. For example you can think of different universities collaborating to perform different computations for the same project. One university might perform computations using gene expression data about a particular population, while another performs computations using data from another population. Importantly each of these universities might use clusters to perform their specific task.
+Again, just like computer clusters, there are many types of architectures that can be rather simple to very complex. For example you can think of different universities collaborating to perform different computations for the same project. One university might perform computations using gene expression data about a particular population, while another performs computations using data from another population. Within one location, each of these universities might use clusters to perform their specific task.
 
```

Comment:
AVOCADO: I am not sure if I understood this right: so this analogy means that within the Grid, there may be multiple locations that use Clusters (because clusters are typically local and grids typically involve multiple locations?)

Reply:
*I am creating a simple sketch that might illustrate this concept. I will send it to you for review when I make it!